functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/BuildConfigurator.java

```diff
@@ -243,7 +252,7 @@ boolean isQuietMode() {
     }
 
     boolean isSkipTestsEnabled() {
-        return skipTests;
+        return skipTests || isSkipTestsSetInPom();
```

Comment:
relates to https://github.com/arquillian/smart-testing/pull/211/files#r144826045

Reply:
For some reason, this doesn't work and we need to explicitly set the skip test property as done above.  

The issue is here, if I remove skipTest configuration from here, https://github.com/hemanik/smart-testing/blob/ec98c148c6e51d0249315319f1e9fbb68b68d08e/functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectBuilder.java#L73, then `skipTests` property works without any changes being made explicitly but `skipITs` doesn't work at all.

If I configure `skipTest` method in the projectBuilder, then I also need to handle the case, when it is set in pom explicitly.

Reply:
> but `skipITs` doesn't work at all.

What do you mean by that? I removed it, then executed [this test](https://github.com/arquillian/smart-testing/pull/211/files#diff-d7b0ca86a6c38fc16017af7fe32281a1R127) and it was green...